src/components/PeerInfo.js

```diff
@@ -83,7 +87,7 @@ const PeerInfo = ({ peers }) => {
   ));
   return (
     <>
-      <Card title={t("Peers")}>{peerTable}</Card>
+      <Card title={t("peers.peers")}>{peerTable}</Card>
```

Comment:
we have a few scenarios where we get `key.key`, just because I named the keys after what's translated. I'm happy to change this to `peers.title` if we think it necessary